catalog/samples/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: "Sample App Catalog"
+permalink: /catalog/samples/
+---
+
+Complete applications that demonstrate how to get things done with Flutter. Each sample app features a few classes or an animation, a layout, or other feature of Flutter. The samples are short, just one file and usually only one or two pages of code. They should easy to try out with your favorite IDE.
+
```

Comment:
Sure, the contribution part can wait until we stabilize the format. But I thought it may be useful to explain that this is just the initial set of samples and solicit ideas about what samples people would like to see in this catalog.